packages/python/plotly/plotly/tests/test_optional/test_px/test_px.py

```diff
@@ -1,65 +1,77 @@
 import plotly.express as px
 import plotly.io as pio
+import narwhals.stable.v1 as nw
 import numpy as np
 import pytest
 from itertools import permutations
 
 
-def test_scatter():
-    iris = px.data.iris()
-    fig = px.scatter(iris, x="sepal_width", y="sepal_length")
+def test_scatter(constructor):
```

Comment:
@FBruzzesi I'm not seeing the benefit to calling `nw.from_native()` and `df.to_native()` in these tests -- it appears the df is simply being wrapped in Narwhals and then unwrapped back to native -- let me know if I'm missing something.

Reply:
I think it's so that in L14 `iris.get_column("sepal_width").to_numpy()` works library-agnostically, regardless of which exact backend `constructor` is

Reply:
Got it, I see it now, thanks. 

I see a few tests further down, though, where it doesn't seem necessary -- what about the following?

- `test_labels`
- `test_line_mode`
- `test_px_templates`
- `test_marginal_ranges`

Reply:
I am addressing those, thanks for spotting them! Replacing all with regex was definitly a mistake 🙈